backend/overlapping-block-marker/IncrementSerializedClassifier.py

```diff
@@ -0,0 +1,26 @@
+import os
+import subprocess
+import sys
+
+# alabno/simple-haskell-marker
+exec_dir = os.path.abspath(os.path.dirname(os.path.abspath(__file__)))
+
+# get list of items in training directory
+dir_content = os.listdir(exec_dir + os.sep + 'training')
+
+# sort the list
+dir_content.sort()
```

Comment:
This is G's work tbh, I recycled it. I believe it's because training files have incremental names (numerically), so sorting them gives you back the files from the oldest to the youngest?
@giuliojiang